package.json

```diff
@@ -28,16 +28,16 @@
   },
   "homepage": "https://github.com/Microsoft/powerbi-visuals-mekkochart#readme",
   "devDependencies": {
+    "@babel/polyfill": "^7.4.4",
```

Comment:
Replace babel by corejs: https://babeljs.io/docs/babel-polyfill

Reply:
Thank you for the feedback, I have made the change.